rmw_fastrtps_shared_cpp/include/rmw_fastrtps_shared_cpp/custom_subscriber_info.hpp

```diff
@@ -41,6 +41,7 @@ typedef struct CustomSubscriberInfo : public CustomEventInfo
   eprosima::fastrtps::Subscriber * subscriber_;
   SubListener * listener_;
   rmw_fastrtps_shared_cpp::TypeSupport * type_support_;
+  const void * type_support_impl_;
```

Comment:
@MiguelCompany here and elsewhere, out of curiosity, why not storing a pointer to the type support struct instead of those of its individual members?

Reply:
@hidmic Avoiding an indirection every time we publish or take improves performance, IMHO

Reply:
Alright, that sounds reasonable.